dev/Dockerfile.protos

```diff
@@ -3,10 +3,11 @@
 # $ docker run --rm -w /app -v $(pwd):/app gen-protos ./dev/generate-protos.sh
 
 FROM ubuntu:20.04
-
-RUN apt-get update --yes
-RUN apt-get install curl unzip --yes
-RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.19.4/protoc-3.19.4-linux-x86_64.zip
-RUN unzip protoc-3.19.4-linux-x86_64.zip -d /root/protoc
 ENV PATH="/root/protoc/bin:${PATH}"
-RUN protoc --version
+RUN apt-get update \
```

Comment:
```suggestion
RUN apt-get update --y \
```

Is `-y` missing?

Reply:
apt-get update doesn't ask anything, so -y no needed

Reply:
apt-get update doesn't ask anything (in this case), so -y no needed